tests/mocha/event_procedure_create_test.js

```diff
@@ -131,12 +130,6 @@ suite.skip('Procedure Create Event', function() {
             this.workspace.id);
       });
 
-      test.skip(
-          'a model is not deleted if no model with a matching ID exists',
-          function() {
-            // TODO: Do we want this to throw? warn? do nothing?
-          });
```

Comment:
This seems like a useful test.  Why not implement it?

Reply:
Do you have any opinions about what the behavior should actually be?

Reply:
Ideally: whatever constrains us the least in the future.  Unfortunately, whether we fail silently or throw an error _some_ dev is going to depend on that and will write code that breaks if we change it—but probably safer to throw.

Is there any precedent to handling similar situations in other events?

Reply:
Ok I think what we actually want this to do is nothing. that's what the other events do. Plus it makes the most sense for having events be shared between workspaces.

Say you have two workspaces, wsA and wsB. When an event X is fired in wsA you want to run it in wsB. But when you run event X, it fires a new event Y in wsB. So your code tries to run Y in wsA. What we want is for that to just fail silently, and not fire an event.

Reply:
So I think that is actually covered by the following test. Does that look correct to you?